app/src/main/java/com/worldsnas/starplayer/di/module/NetworkModule.kt

```diff
@@ -0,0 +1,75 @@
+package com.worldsnas.starplayer.di.module
+
+import android.app.Application
+import com.google.gson.FieldNamingPolicy
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import com.worldsnas.starplayer.model.api.WebService
+import dagger.Module
+import dagger.Provides
+import dagger.Reusable
+import okhttp3.Cache
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+import javax.inject.Singleton
+
+@Module
+object NetworkModule {
+
+    @JvmStatic
+    @Provides
+    fun provideWebService(retrofit: Retrofit): WebService {
```

Comment:
har feature i mitone/behtare  service e khodesho dashte bashe 
pas niazi be inja dashtanesh nist